src/pages/Blog/index.tsx

```diff
@@ -1,13 +1,22 @@
-import React, { Fragment } from 'react'
+import React, { Fragment, useEffect } from 'react'
 import Layout from '../../components/Layout'
 import BlogPage from '../../components/blogComponent/Blog'
 import { BlogPosts } from '../../mockS/mockBlogData'
+import { useSelector, useDispatch } from 'react-redux'
+import { setBlogs } from '../../redux/reducers/blogSlice'
+import { RootState } from '../../redux/store/configureStore'
 
 const Blog: React.FC = () => {
+  const dispatch = useDispatch()
```

Comment:
let's give this a descriptive name just so we know what action we are dispatching here.